chmodzip.d

```diff
@@ -74,11 +73,10 @@ L1:
 
         foreach (member; members)
         {
-            if (de.name == member && ((de.externalAttributes >> 16) & octal!7777) != newattr)
+            if (de.name == member && (de.fileAttributes & octal!7777) != newattr)
             {
                 changes = true;
-                de._madeVersion = 0x317; // necessary or linux unzip will ignore attributes
-                de.externalAttributes = (de.externalAttributes & ~(octal!7777 << 16)) | (newattr << 16);
+                de.fileAttributes = de.fileAttributes & ~octal!7777 | newattr;
```

Comment:
I'm not sure about these changes - on one hand if the script wasn't building no idea if it was actually used any more, on the other hand this needs to work correctly so that unzipping .zip files on POSIX results in the correct attributes being set for files, and I doubt this is tested automatically anywhere right now. @MartinNowak ?